starter/source/tools/sensor/read_sensor_disp.F

```diff
@@ -158,8 +152,10 @@ SUBROUTINE READ_SENSOR_DISP(SENSOR_PTR ,SENS_ID  ,
  2000 FORMAT(
      . 5X,'   NODE 1 ID . . . . . . . . . . . . . . .=',I10/
      . 5X,'   NODE 2 ID . . . . . . . . . . . . . . .=',I10/
+     . 5X,'   DESACTIVATION FLAG. . . . . . . . . . .=',I10/
```

Comment:
There is typo : desactivation => deactivation.

I am not sure what do you mean by "MIN TIME INTERVAL TO ACTIVATE".